src/utils/index.js

```diff
@@ -1,12 +1,13 @@
-// This file contains utility functions that are exposed game-side
+/** This file contains utility functions that are exposed game-side */
 
 /**
   * Choose a random number between a and b.
   * @example randRange([a, b], bipolar)
   * @example randRange(a, b, bipolar)
-  * @arg {number} a
-  * @arg {number} b
-  * @arg {boolean} [bipolar] - If true, then the returned value will be inverted half of the time
+  * @global
+  * @arg     {number}  a
+  * @arg     {number}  b
+  * @arg     {boolean} [bipolar=false] Should the returned value will be inverted half of the time
   * @returns {number}
   */
 randRange = function(a, b, bipolar) {
```

Comment:
@yofreke  `rollInt` is supposed to be inclusive on both sides, it's not Math.random(), it's intended to choose an int in this range, inclusively. Though I hadn't tested either with invalid params, I can see how including 1 in that case would break code you want to always floor to 0. I understood the order (min, max) to be a standard across math and programming.